package-lock.json

```diff
@@ -9399,8 +9399,7 @@
           "version": "3.0.0",
           "resolved": "https://registry.npmjs.org/get-stream/-/get-stream-3.0.0.tgz",
           "integrity": "sha1-jpQ9E1jcN1VQVOy+LtsFqhdO3hQ=",
-          "dev": true,
-          "optional": true
+          "dev": true
```

Comment:
Do you mind reverting the changes to this file? One of our dependencies causes it to change randomly, and we'd like to avoid potential conflicts by changing it unncessarily.